web/src/lib/stores/albums.store.ts

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const searchAlbum = writable<string>('');
```

Comment:
Why use a store? To persist the value after navigation? Then I think this should be added to the `albumViewSettings` store, otherwise it can be a regular prop.

Reply:
It's to share value between `albums-list` and `albums-control` which are both child components of `+page.svelte`. I prefer to use a store for that kind of use-case